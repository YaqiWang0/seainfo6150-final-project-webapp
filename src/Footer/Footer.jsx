import React from "react"
import style from "./Footer.module.css"

const Footer = () => {
    return (
        <div className={style.container}>
            <div>Author: Yaqi Wang</div>
            <div>@All the images and content are downloaded or fetched from Internet.</div>
        </div>
    )
}
export default Footer