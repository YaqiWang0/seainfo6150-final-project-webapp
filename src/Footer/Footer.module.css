.container {
    margin: 0 auto;
    width: 80%;
    background-color: rgb(0, 0, 0);
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
    color: rgba(255, 255, 255, 0.5);
    font-size: 1em;
    padding: 1em;
}