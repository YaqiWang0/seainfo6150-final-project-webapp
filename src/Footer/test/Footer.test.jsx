import React from "react";
import Footer from "../Footer";

describe("Footer test", () => {

    it('render successfully', function () {
        const {container} = render(
            <Footer/>
        );
        expect(container).toMatchSnapshot();
    })
})