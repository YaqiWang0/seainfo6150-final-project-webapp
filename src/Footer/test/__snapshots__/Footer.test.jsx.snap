// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer test render successfully 1`] = `
<div>
  <div
    class="container"
  >
    <div>
      Author: Yaqi Wang
    </div>
    <div>
      @All the images and content are downloaded or fetched from Internet.
    </div>
  </div>
</div>
`;
